### DISCUSSION TITLE
How to subscribe to the "swipe back event" on iOS?

### DISCUSSION THREAD

**Discussion by RsZoli** [dotnet/maui#13287] [Q&A]
I would like to subscribe to the "swipe back event" of the InteractivePopGestureRecognizer on iOS platform, just like the OnBackButtonPressed event on Andorid.

Please help my noob self, i googled it for hours already, but everyone seem to just want to disable the event all together. :(

Thank you very much!

**Comment by EdwardMcFlurry** [comment]
I’m also curious 🧐

**Comment by Xyncgas** [comment]
In my experience you can do this in blazor by listening to the navigation related events
in XAML idk how but you can use Blazor in MAUI
if MAUI doesn't work somehow then you still have PWA with blazor that can be published to IOS store
